launch/launch/utilities/file_path.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Rover Robotics, c/o Dan Rose
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import typing
+
+# Type of a filesystem path
+FilePath = typing.Union[str, bytes, os.PathLike]
```

Comment:
I would rename this as `FilePathType`.

Reply:
I don't like that because none of the type annotation types have `Type` in them, to keep them succinct. I did make it a type var to better match e.g. `typing.AnyStr`.

Reply:
> I don't like that because none of the type annotation types have Type in them, to keep them succinct. I did make it a type var to better match e.g. typing.AnyStr.

I think that in the case of `typing` is clear, because that module only has type annotation types.
Now, the file has `_type` suffix in the name, so it's clear that it only contains a type definition.

If you also would like to add `FilePath` to `utilities` `__all__` dictionary, please rename it in that case as `FilePathType` (or re-export a renamed version).

----

You can omit this last suggestion if you want.